test/robosherlock/test.cpp

```diff
@@ -0,0 +1,37 @@
+#include <string>
+#include <gtest/gtest.h>
+#include "rs/utils/RSAnalysisEngineManager.h"
+#include "rs/utils/RSAnalysisEngine.h"
+#include <rs/utils/common.h>
+#include <rs/types/all_types.h>
+#include <rs/scene_cas.h>
+
+#include <pcl/point_types.h>
+#include <ros/ros.h>
+
+class ExampleTest : public ::testing::Test
+{
+protected:
+    virtual void SetUp()
+    {
+        rs::common::getAEPaths("test",engineFile);
+        engine.init(engineFile);
+    }
+
+    virtual void TearDown(){}
+    std::string engineFile;
+    RSAnalysisEngine engine;
+
```

Comment:
could we use RSPipelineManager  so we need only one xml file for all tests? In the SetUp() function one could initialize it and then set the desired pipeline using vectors of strings.

Reply:
cool, I will check this out asap.